chain/solana/provider/ctf_provider.go

```diff
@@ -0,0 +1,283 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/avast/retry-go/v4"
+	sollib "github.com/gagliardetto/solana-go"
+	solrpc "github.com/gagliardetto/solana-go/rpc"
+	chain_selectors "github.com/smartcontractkit/chain-selectors"
+	solCommonUtil "github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/common"
```

Comment:
maybe change this to `sol_utils` or `solutils`? to keep the casing consistent
```suggestion
	solCommonUtil "github.com/smartcontractkit/chainlink-ccip/chains/solana/utils/common"
```